.github/workflows/check-pull-request-title-and-description.yml

```diff
@@ -0,0 +1,22 @@
+name: Check-pull-request-title-and-description
+
+on: 
+  pull_request:
+    types: [opened, synchronize, reopened, edited]
+jobs:
+  Check-commit-messages:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+
+      - name: Check the pull request
+        uses: mristin/opinionated-commit-message@v2.3.1
+        with:
+          path-to-additional-verbs: additional-verbs-in-imperative-mood.txt
+
+      - name: READ HERE ON FAILURE FOR MORE INSTRUCTIONS
+        if: ${{ failure() }}
+        run: |
+          echo \
+            "The title and the description of your pull request do not fit our style guide." \
```

Comment:
Maybe we don't talk about a style guide until we really add one. WDYT?

Reply:
Thanks for spotting it! I copy-pasted the script and adapted it. Though I removed the details from the other project, I did not pay attention to the actual message.

Yes, I agree, let's wait for the guidelines to be defined. The guideline should also be linked in the message.